packages/components/src/templates/next/layouts/Homepage/Homepage.stories.tsx

```diff
@@ -603,6 +603,28 @@ const generateArgs = ({
         url: "/",
         label: "More ways to contact us",
       },
+      {
+        type: "logocloud",
+        title: "Our partners",
+        images: [
+          {
+            src: "https://isomer-user-content.by.gov.sg/16/25863c2f-61e7-4d54-bb36-e7cfa1490b64/ogp-logo.svg",
+            alt: "open government products logo",
+          },
+          {
+            src: "https://isomer-user-content.by.gov.sg/85/06d6b95f-fc6b-418d-8b08-9efa88790822/GovTech%20Inline%20Logo_V3_no%20bg.gif",
+            alt: "govtech logo",
+          },
+          {
+            src: "https://isomer-user-content.by.gov.sg/1/713bc29a-4ba6-4e17-b3a0-43b5a88a4572/stb-logo.svg",
+            alt: "singapore tourism board logo",
+          },
+          {
+            src: "https://isomer-user-content.by.gov.sg/24/b6286d60-5e3c-430a-a9e6-9a4c1dce6544/togsgcares_logo.png",
+            alt: "sg cares logo",
+          },
+        ],
```

Comment:
ps, can we don't use stuff from `isomer-user-content`? since it contains the UUID + might be blocked by WAF when chromatic is running